spec.html

```diff
@@ -13792,15 +13809,26 @@ <h1>Static Semantics: MV</h1>
 
       <emu-clause id="sec-numericvalue" type="sdo" aoid="NumericValue">
         <h1>Static Semantics: NumericValue</h1>
-        <emu-grammar>NumericLiteral :: DecimalLiteral</emu-grammar>
+        <emu-grammar>NumericLiteral :: NonDecimalIntegerLiteral</emu-grammar>
         <emu-alg>
-          1. Return the Number value that results from rounding the MV of |DecimalLiteral| as described below.
+          1. Let _mv_ be the MV of |NonDecimalIntegerLiteral|.
+          1. Return 𝔽(_mv_).
         </emu-alg>
-        <emu-grammar>NumericLiteral :: NonDecimalIntegerLiteral</emu-grammar>
+
+        <emu-grammar>NumericLiteral :: DecimalLiteral</emu-grammar>
         <emu-alg>
-          1. Return the Number value that results from rounding the MV of |NonDecimalIntegerLiteral| as described below.
+          1. If |DecimalLiteral| has more than 20 significant digits, then
+            1. Let _lit_ be an implementation-defined choice of:
+              * a literal produced from |DecimalLiteral| by replacing each significant digit after the 20th with a `0` digit
+              * a literal produced from |DecimalLiteral| by replacing each significant digit after the 20th with a `0` digit and then incrementing the literal at the 20th significant digit position
+            1. NOTE: Because of the limited precision of the Number type, the Number value for the MV of |DecimalLiteral| will always be equal to the Number value for the MV of at least one of those two literals.
```

Comment:
The target of "those" was not immediately obvious here.

```suggestion
            1. NOTE: Because of the limited precision of the Number type, the Number value for the MV of |DecimalLiteral| will always be equal to the Number value for the MV of at least one of the permitted literals.
```

Reply:
Well, @bakkot suggested "those two literals", so I'll let you two fight it out. Another possibility would be "the above two literals".

Reply:
"the above two literals" works for me.

Reply:
Done.